src/blueapi/config.py

```diff
@@ -35,6 +35,11 @@ class LoggingConfig(BlueapiBaseModel):
     level: LogLevel = "INFO"
 
 
+class FastApiConfig(BlueapiBaseModel):
```

Comment:
Think this could have a more generic name, nothing ties it to FastAPI, maybe `RestConfig`?

Reply:
good point, will change.